mapfishapp/src/main/webapp/app/addons/goto/manifest.json

```diff
@@ -7,7 +7,8 @@
         "projections": [{
             "srs": "EPSG:4326", 
             "name": "WGS 84",
-            "labels": ["Longitude", "Latitude"]
+            "labels": ["Longitude", "Latitude"],
+            "decimalPrecision": 10
```

Comment:
10 does not look like a relevant default value here.
6 would better suit the general use case IMO.

config.json should also be updated.